src/providers/html5/html5.ts

```diff
@@ -41,17 +41,7 @@ export default function Html5Provider(Player: any) {
 		 * @returns The video is ready
 		 */
 		waitUntilVideoIsReady(): Promise<any> {
-			return new window.Promise<void>((resolve) => {
```

Comment:
I am thinking maybe keep this as an option. I don't see much use but maybe someone wants to show ready player only when preload is complete. Maybe one use case is if you don't use poster image and show first frame instead. Then maybe it would be better user experience show loading indicator and no play button until metadata is ready and first frame can be displayed as a poster.

Reply:
Yes the initial condition is ok, I just updated it to work without preloading